sormas-api/src/main/java/de/symeda/sormas/api/utils/InfoProvider.java

```diff
@@ -49,7 +49,7 @@ public static synchronized InfoProvider get() {
 	 * @return
 	 */
 	public String getMinimumRequiredVersion() {
-		return "1.52.0";
+		return "1.53.0";
```

Comment:
Not sure this is needed.

Reply:
It's written [here](https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/GUIDE_ADD_NEW_FIELD.md). Why shuold it not be necessary?

Reply:
As I know this version number should be changed once per release, I assume before doing the actual release.

Reply:
That was forgotten frequently. So it's no harm to do it here already, or?

Reply:
You can keep it if you want